src/service.ts

```diff
@@ -95,7 +94,7 @@ async function resolveConfigNoCache(
     );
   }
 
-  return { ...config, filepath };
+  return { ...(config ?? {}), filepath };
```

Comment:
Previously, if no config was found but `PRETTIERD_DEFAULT_CONFIG` was set but not resolved `prettierd` would fail due to it trying to spread `undefined`.

Reply:
This works:

```
％ node
Welcome to Node.js v16.5.0.
Type ".help" for more information.
> {...undefined, x: 1}
{ x: 1 }
```

(TypeScript wouldn't allow it otherwise)